lib/Doctrine/ORM/Internal/Hydration/ScalarColumnHydrator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Internal\Hydration;
+
+/**
+ * Hydrator that produces one-dimensional array "graph" that is often (not always) interchangeable
+ * with the corresponding object graph for read-only access.
```

Comment:
That comment is misleading, there is really no graph here, because you cannot select more than 1 scalar. In order to have something that would look like a graph, you would need to have at least 2 entities selected.

Reply:
```php
 * Hydrator that produces one-dimensional array that is often (not always) interchangeable
 * with the corresponding object for read-only access.
```

Is this OK ?

Reply:
> that is often (not always) interchangeable with the corresponding object for read-only access.

I think you can remove this part of the comment, to understand why it does not make sense, you can read my recent answer on this SO question: https://stackoverflow.com/questions/30050886/doctrine-orm-hydrate-scalar-vs-hydrate-array/68795117#68795117